chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+if ! [[ -r `pwd` ]]; then
+    echo "Directory is not readable."
+    exit 255
+elif [[ "$#" -lt 2 ]]; then
+    echo "At least 2 args are expected"
+    exit 1
+elif [[ "$#" -gt 2 ]]; then
+    echo "Too many args, only 2 args are expected"
+    exit 1
+elif [[ -e $1 && -f $2 ]]; then
+    cat $1 $2
+elif [[ ! -e $1 && -e $2 ]]; then
+    echo file $1 not exist, it will be created. File $2 here:
+    cat $2
+    `openssl rand -base64 20 > $1 && chmod 700 $1`
```

Comment:
So here and below you can see that you repeated in code the same behavior 4 times, it is not cool, because if you will need to add more changes, you will need to do it 4 times instead of 1. So we always try to implement code to `don't repeat yourself` - https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
Added a function

Reply:
Still a lot of repetitions